package.json

```diff
@@ -99,8 +99,8 @@
     "react-dom": "^18.2.0",
     "stylelint": "15.x",
     "stylelint-config-standard-scss": "10.x",
-    "type-fest": "^3.10.0",
-    "typescript": "~4.9.5"
+    "type-fest": "4.x",
+    "typescript": "~5.1.6"
```

Comment:
Do we want to fix on TS 5.1, vs allowing 5.x?  I see that 5.2 is currently in beta, but don't know how big of a deal TS "minor" version updates typically are.